package.json

```diff
@@ -8,14 +8,23 @@
     "compile": "lerna run --scope @openmsupply-client/* --parallel tsc",
     "prepare": "husky install",
     "pre-commit-lint": "yarn compile && yarn lint-staged",
-    "test": "jest --config ./jest.config.js"
+    "test": "jest --config ./jest.config.js",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
```

Comment:
unsure if we should use a `@openmsupply-client/ui` package with all the building blocks and just use story book for that..

But on the other hand, it is a super nice way for 'visual testing' - easily be able to switch through components and check that strings fit etc.

Reply:
I really like the idea of a ui package but do worry that updating, bumping revisions, updating usages.. all too much hassle at the mo. maybe later?
agree though, that storybook does help with checking out components which are built of items in the ui package, and so it is good to have it at this level